docs/guide/field_type_reference.md

```diff
@@ -1214,7 +1214,9 @@ $keywordValue = new Value( "php5,css3,html5" );
 
     Landing Page Field Type is only used in the Landing Page Content Type that is included in eZ Enterprise.
 
-    The structure of the Landing Page Content Type should not be modified, as it may cause errors.
+    !!! caution
+    
+        The structure of the Landing Page Content Type should not be modified, as it may cause errors.
```

Comment:
It is more on pre-defined FieldTypes within the LandingPage contentType. Of course you are able to add new FieldTypes to the LandingPage but don't change any existing identifier (name, description,page)